packages/apps/docs/src/pages/build/guides/election-dapp-tutorial/index.md

```diff
@@ -1,83 +1,91 @@
+
+- Create and fund accounts.
+- Leverage keysets and capabilities to manage permissions.
+- Interact with the blockchain directly from your contract code.
+- Read data from the blockchain state for free.
+- Sign and send transactions that require fee payments.
+- Deploy and update a smart contract.
+- Use a gas station to pay fees on behalf of application users.
+
+Some of the topics covered in this workshop are similar to topics introduced in the _Quick start_.
+For example, you’ll set up a development network (devnet) running on your local computer in a Docker container and sign transactions using public keys and Chainweaver accounts.
+However, the tutorials in the workshop also touch on additional topics such as:
+
+- How to develop smart contracts that use formal verification.
+- Options available for signing transactions.
+- Where you can find additional tooling for blockchain development, including the Kadena JavaScript client library, Chainweaver, the Pact command-line executable, and more.
+
+## Workshop roadmap
+
+This workshop consists of multiple tutorials that you should complete in order. 
+Each tutorial provides some starter code and guides you to completing the tasks necessary to build the Election application to run on the Kadena blockchain.
+
+For the first tutorial, you’ll clone a repository with a working front-end for the election application that stores candidates and votes in memory. 
+After that, you’ll set up a local development network and your development workspace to begin building the blockchain back-end for the election website. 
+The workshop repository contains branches with starter code for every tutorial, so you can compare your solution with the finished code at every step along the way. 
+
+ * [Get started](/build/guides/election-dapp-tutorial/01-getting-started)
+ * [Prepare your workspace](/build/guides/election-dapp-tutorial/02-running-devnet)
+ * [Admin account](/build/guides/election-dapp-tutorial/03-admin-account)
+ * [Namespaces](/build/guides/election-dapp-tutorial/04-namespaces)
+ * [Keysets](/build/guides/election-dapp-tutorial/05-keysets)
+ * [Smart contract](/build/guides/election-dapp-tutorial/06-smart-contract)
+ * [Nominate candidates](/build/guides/election-dapp-tutorial/07-nominate-candidates)
+ * [Voting](/build/guides/election-dapp-tutorial/08-voting)
+ * [Gas station](/build/guides/election-dapp-tutorial/09-gas-station)
+
+Stay tuned for future editions of this workshop and for more intermediate and advanced tutorials  covering additional topics, such as:
+
+ * Deploy to test network with the functional programming approach
+ * Develop a continuous integration pipeline for deploying your application
+
+To suggest topics you’d like to see covered in future tutorials and workshops, submit a [documentation request]().
+If you'd like to contribute content—either in the form of a new tutorial or to fix issues—see our [contributor's guide]().
```

Comment:
Personally I would put spaces arounds the dashes, I find it more readable.

Could also be just me, I'm not a native speaker :)

Reply:
The dashes render as em dashes (—) so they typically don't use extra spaces, but in the raw markdown they look a little cramped.